rholang/src/main/scala/rholang/rosette/Roselang.scala

```diff
@@ -175,36 +177,37 @@ extends AllVisitor[VisitorTypes.R,VisitorTypes.A] {
       }
 
       val ptrnTermList = p.listcpattern_.asScala.toList
-      val bindingsComponents = ptrnTermList map {
+      val bindingsComponents : List[Option[List[StrTermCtxt]]] = ptrnTermList map {
```

Comment:
What's the difference here between None and an empty list of StrTermCtxts?

Reply:
It was always producing that type.

The PR that I'm working on will get rid of Option.